financialaid/serializers.py

```diff
@@ -0,0 +1,54 @@
+"""
+Serializers from financialaid
+"""
+import datetime
+from django.shortcuts import get_object_or_404
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+from rest_framework.fields import IntegerField, FloatField, CharField
+
+from courses.models import Program
+from dashboard.models import ProgramEnrollment
+from financialaid.backend import determine_tier_program, determine_auto_approval
+from financialaid.models import FinancialAid
+
+
+class FinancialAidSerializer(serializers.Serializer):
+    """
+    Serializer for Financial Aid objects
+    """
+    original_income = FloatField(min_value=0)
+    original_currency = CharField()
+    program_id = IntegerField()
+
+    def validate(self, data):
+        """
+        Validators for this serializer
+        """
+        data["program"] = get_object_or_404(Program, pk=data["program_id"])
+        if not data["program"].financial_aid_availability:
+            raise ValidationError("Financial aid not available for this program.")
+        if not ProgramEnrollment.objects.filter(program=data["program"], user=self.context["request"].user).exists():
```

Comment:
The way the code is now is bad design and changing it to use a permission class is a trivial change IMO. But I don't think it's worth holding up the review as long since we test this